src/universal_transfer_operator/data_providers/database/snowflake.py

```diff
@@ -15,10 +15,22 @@
 from universal_transfer_operator.universal_transfer_operator import TransferIntegrationOptions
 
 
+@attr.define
+class SnowflakeOptions(TransferIntegrationOptions):
+    """
+    Snowflake load option for naive transfer.
+    """
+
+    file_options: dict = attr.field(factory=dict)
+    copy_options: dict = attr.field(factory=dict)
+    storage_integration: str | None = attr.field(default=None)
+
+
 class SnowflakeDataProvider(DatabaseDataProvider):
     """SnowflakeDataProvider represent all the DataProviders interactions with Snowflake Databases."""
 
     DEFAULT_SCHEMA = SNOWFLAKE_SCHEMA
+    OPTIONS_CLASS = SnowflakeOptions
```

Comment:
What would happen if there are multiple option classes available?

Reply:
Can point out a use case where this would happen?